### ISSUE TITLE
Using `yield_to` with  signals that take parameters

### ISSUE THREAD

**Issue by bitwes** [bitwes/Gut#258]
I got engine error messages when I added a parameter that a signal being  used with `yield_to`,

```
 Error calling method from signal 'fetch_done': 'Control(gut.gd)::_yielding_callback': Method expected 1 arguments, but called with 2..
```

**Comment by bitwes** [comment]
Adding an arbitrary number of optional parameters to the callback method worked as a hack for my personal project.  This would need to be formalized with tests and such, but it appeared to work:
``` gdscript
# gut.gd
func _yielding_callback(from_obj=false, p1=null, p2=null, p3=null, p4=null, p5=null):
```

**Comment by nightblade9** [comment]
I have this problem too. To clarify, this will be fixed in the upcoming Gut 7.2.0 release, right? It looks like it's already closed/tagged.

**Comment by bitwes** [comment]
Correct.
